src/react-components/webpack.prod.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
   devtool: "source-map",
   entry: {
     app: [
-      "./src/index.ts"
+      "./src/index.ts", 
+      // "./src/styles.scss"
```

Comment:
so this interferes with importing design system in the consumer app.  I was able to build it without, but ... I'm kind of confused about it

Reply:
From this and the removal of `import './styles.scss';`, this means that the scss is not available anymore for the consumer app, right?

Reply:
because it's being copied over to lib and dist, and then imported directly, it seems to still work.